addons/ovm/examples/ovm.tx

```diff
@@ -0,0 +1,93 @@
+addon "ovm" {
+    l1_rpc_api_url = "http://localhost:9999"
+    working_dir = "/tmp/ovm"
+}
+addon "evm" {}
+
+variable "l1_deployment_addresses" {
+    value = {
+        "AddressManager": "0x2021fD587B7Bc8EF1Aa8370aE2f91516C6D29648",
+        "DisputeGameFactory": "0x06a0Bf963662754C68612af8e6229b726C826E7F",
+        "DisputeGameFactoryProxy": "0xE163428C3A1e539586b9d01663C4812cF11e221f",
+        "L1CrossDomainMessenger": "0xD9E1f575Ca0C2612ae1BE934cc3dbD9970D2DdeC",
+        "L1CrossDomainMessengerProxy": "0x9D6F091239552bB98A22FF695f63bDF282FE8a7b",
+        "L1ERC721Bridge": "0xDf066f6930E6e87bb93A38F573D68e65b8da3E39",
+        "L1ERC721BridgeProxy": "0x87EaE5Ea46dEe0e3e8993036E284fA4Aaf15A641",
+        "L1StandardBridge": "0x50d9AEB686810E9cf4979bfCAA3990fCa74e0830",
+        "L1StandardBridgeProxy": "0x0C5C3aF1508278cbC536827a1cCb2f69195b7068",
+        "L2OutputOracle": "0x0fA6ac773c5E2fcaFb6aa8825245f266b810D0BF",
+        "L2OutputOracleProxy": "0xd175D46c2016E8f4aC49db91d929134fb38DEDAE",
+        "Mips": "0xb69a4f34fd8170FA3050297F8F132682822Bbb6E",
+        "OptimismMintableERC20Factory": "0xE3E0228213325f4d89044bea2f5236C48d16ad6f",
+        "OptimismMintableERC20FactoryProxy": "0x499467B8984877dA39Eca9D933545adcc95FB967",
+        "OptimismPortal": "0x0D8D2f43375e4c76d0AC5a6b48E640c58F549278",
+        "OptimismPortalProxy": "0x498D7D81fe926d5C5b98435C5acfA7e6B3319940",
+        "PreimageOracle": "0xF25e2d47Ac5B668150a2a023051715B5aA0C904B",
+        "ProtocolVersions": "0x6226b256912843C624B163b1CC9dB13D96894b03",
+        "ProtocolVersionsProxy": "0x1cA58C594C4Db979e0aaD6B821018074CBFa3d48",
+        "ProxyAdmin": "0xa869413806d4496BE149896d0c01f5b78e790d23",
+        "SafeProxyFactory": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
+        "SafeSingleton": "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
+        "SuperchainConfig": "0xdD6191EE7b7aeCca6f14Ef2bCF76183467D6ED0E",
+        "SuperchainConfigProxy": "0x3389EA8C39393D0073920EF23e9f4F73C3930637",
+        "SystemConfig": "0x5fEE6D8c69e309d636A40492CD10950fA9422E69",
+        "SystemConfigProxy": "0x677209694761A0B010c1C94503b0C310f0e6Bd21",
+        "SystemOwnerSafe": "0x44e5a74148BFAf7713Bf3bA050896982e69bBFea"
```

Comment:
Will developers have to provide all these addresses?

Reply:
For now, yes. These are the deployments from the L1 terraforming.

Eventually we will even have these be integrated as part of the command, or they'll be a standalone runbook that you can call.